### PR TITLE
kucoin - params `chain` fix `fetchTransactionFee`

### PR DESCRIPTION
**Pull request by AlgoQ**
#15692
# Testing
## Python
`python3 examples/py/cli.py kucoin fetchTransactionFee "USDT" '{"network":"BEP20"}'`
`python3 examples/py/cli.py kucoin fetchTransactionFee "USDT" '{"network":"bsc"}'`
`python3 examples/py/cli.py kucoin fetchTransactionFee "USDT" '{"network":"BSC"}'`
```
Python v3.10.6
CCXT v2.1.79
kucoin.fetchTransactionFee(USDT,{'network': 'BEP20'})
{'deposit': {},
 'info': {'code': '200000',
          'data': {'availableAmount': '0.00000094',
                   'chain': 'BEP20',
                   'currency': 'USDT',
                   'innerWithdrawMinFee': '0',
                   'isWithdrawEnabled': False,
                   'limitBTCAmount': '1.00000000',
                   'precision': 8,
                   'remainAmount': '16515.27663088',
                   'usedBTCAmount': '0.00000000',
                   'withdrawMinFee': '5',
                   'withdrawMinSize': '10'}},
 'withdraw': {'USDT': 5.0}}
```

### CODE REVIEW

File: js/kucoin.js [hunk 1]
```diff
@@ -799,7 +801,7 @@ module.exports = class kucoin extends Exchange {
         let network = this.safeStringUpper (params, 'network');
```
Comment (carlosmiei) [inline js/kucoin.js:801]: @AlgoQ here I was suggesting using `safeStringUpper2` and accept both `chain` and `network`

**Comment by carlosmiei** [comment]
LGTM ✅ 
- fixes https://github.com/ccxt/ccxt/issues/15692

**Comment by carlosmiei** [comment]
@hosseinab yes it should, did you find any issue with it?

**Comment by hosseinab** [comment]
@carlosmiei I updated ccxt to the latest version but it doesn't work

**Comment by carlosmiei** [comment]
@hosseinab can you share with us a verbose log please?

**Comment by hosseinab** [comment]
@carlosmiei 

```
fetch Request: kucoin GET https://api.kucoin.com/api/v1/withdrawals/quotas?currency=STARLY&chain=BEP20 RequestHeaders: {'KC-API-KEY-VERSION': '2', 'KC-API-KEY': '*', 'KC-API-TIMESTAMP': '1668696132175', 'KC-API-PASSPHRASE': 'BIGu/*=', 'KC-API-SIGN': '*=', 'KC-API-PARTNER-SIGN': 'nhQVuXp+*=', 'KC-API-PARTNER': 'ccxt', 'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive'} RequestBody: None

fetch Response: kucoin GET https://api.kucoin.com/api/v1/withdrawals/quotas?currency=STARLY&chain=BEP20 200 ResponseHeaders: {'Date': 'Thu, 17 Nov 2022 14:42:10 GMT', 'Content-Type': 'application/json', 'Content-Length': '41', 'Connection': 'keep-alive', 'Set-Cookie': '*=*/*/*+*/*; Expires=Thu, 24 Nov 2022 14:42:10 GMT; Path=/, AWSALBCORS=*/iwZs8EYb7vmOe6w6b+*/*; Expires=Thu, 24 Nov 2022 14:42:10 GMT; Path=/; SameSite=None; Secure, __cf_bm=*.i128-1668696130-0-ATyq2HQEgKfDmJetA5LgMagZgz+YByyuBznfZ+*+PeQbjYio=; path=/; expires=Thu, 17-Nov-22 15:12:10 GMT; domain=.kucoin.com; HttpOnly; Secure; SameSite=None, __cfruid=*-1668696130; path=/; domain=.kucoin.com; HttpOnly; Secure; SameSite=None', 'Strict-Transport-Security': 'max-age=9800; includeSubDomains; preload', 'CF-Cache-Status': 'DYNAMIC', 'Server': 'cloudflare', 'CF-RAY': '76b93b3d3b629bac-FRA'} ResponseBody: {"code":"900014","msg":"Invalid chainId"}
```

**Comment by carlosmiei** [comment]
@hosseinab can you please double-check your ccxt version, I just tested and seems to be working fine:

```
 p kucoin fetchTransactionFee "STARLY" '{"chain":"BEP20"}' --verbose
Python v3.10.8
CCXT v2.1.89
kucoin.fetchTransactionFee(STARLY,{'chain': 'BEP20'})

fetch Request: kucoin GET https://api.kucoin.com/api/v1/withdrawals/quotas?currency=STARLY&chain=bsc RequestHeaders: {'KC-API-KEY-VERSION': '2', 'KC-API-KEY': '62fbaf62bc95360001717112', 'KC-API-TIMESTAMP': '1668697456928', 'KC-API-PASSPHRASE': 'CKSoDx9p/PZN1GmbWoVtJqzhE73BJh/grLDBCc28j3I=', 'KC-API-SIGN': 'se9xa/8wTCfAKVSkYG1GHb5jDJU7KPyqtaMtKwxD3KU=', 'KC-API-PARTNER-SIGN': '0u5coZQ5xKtMCFvVA5A3CltrUWnKg1uOptL/YSWR3uQ=', 'KC-API-PARTNER': 'ccxt', 'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate'} RequestBody: None

fetch Response: kucoin GET https://api.kucoin.com/api/v1/withdrawals/quotas?currency=STARLY&chain=bsc 200 ResponseHeaders: {'Date': 'Thu, 17 Nov 2022 15:04:17 GMT', 'Content-Type': 'application/json', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'Set-Cookie': 'AWSALB=Q1RgLcy3+qaHVd9XXI6a0+90tc1zc6quzOXRSnHtvK03R9TtxmowJxWGynmBdrsBap/vWWLZ948DMGigDSwbrvkEdtRsNWJ4uOrsPxcZY4NwjfcOu3yumbmUdDGw; Expires=Thu, 24 Nov 2022 15:04:17 GMT; Path=/, AWSALB=Q1RgLcy3+qaHVd9XXI6a0+90tc1zc6quzOXRSnHtvK03R9TtxmowJxWGynmBdrsBap/vWWLZ948DMGigDSwbrvkEdtRsNWJ4uOrsPxcZY4NwjfcOu3yumbmUdDGw; Expires=Thu, 24 Nov 2022 15:04:17 GMT; Path=/', 'Strict-Transport-Security': 'max-age=9800; includeSubDomains; preload', 'CF-Cache-Status': 'DYNAMIC', 'Server': 'cloudflare', 'CF-RAY': '76b95ba27b9903ca-LIS', 'Content-Encoding': 'gzip'} ResponseBody: {"code":"200000","data":{"currency":"STARLY","limitBTCAmount":"1.00000000","usedBTCAmount":"0.00000000","remainAmount":"662251.65562913","availableAmount":"0","withdrawMinFee":"4.5","innerWithdrawMinFee":"0","withdrawMinSize":"20","isWithdrawEnabled":true,"precision":8,"chain":"BEP20"}}
{'deposit': {},
 'info': {'code': '200000',
          'data': {'availableAmount': '0',
                   'chain': 'BEP20',
                   'currency': 'STARLY',
                   'innerWithdrawMinFee': '0',
                   'isWithdrawEnabled': True,
                   'limitBTCAmount': '1.00000000',
                   'precision': 8,
                   'remainAmount': '662251.65562913',
                   'usedBTCAmount': '0.00000000',
                   'withdrawMinFee': '4.5',
                   'withdrawMinSize': '20'}},
 'withdraw': {'STARLY': 4.5}}
```

**Comment by hosseinab** [comment]
@carlosmiei 
i checked ccxt version as belew :
PS C:\Users\Hosein\PycharmProjects\***> pip show ccxt
Name: ccxt
Version: 2.1.89
Summary: A JavaScript / Python / PHP cryptocurrency trading library with support for 130+ exchanges
Home-page: https://ccxt.com
Author: Igor Kroitor
Author-email: igor.kroitor@gmail.com
License: MIT
Location: c:\users\hosein\pycharmprojects\pythonproject\venv\lib\site-packages
Requires: aiodns, aiohttp, certifi, cryptography, requests, setuptools, yarl
Required-by: 

But this merge still doesn't work

**Comment by carlosmiei** [comment]
Hello @hosseinab, I'm sorry you're right, we had some issues with the build so the publishing process did not succeed. The build is ok now so I'd expect a new release in +-45min. Sorry for the inconvenience!

**Pull request merged**